CMakeLists.txt

```diff
@@ -77,6 +79,8 @@ endif()
 
 cmake_print_variables(WITH_KRNLMON)
 cmake_print_variables(WITH_OVSP4RT)
+cmake_print_variables(LNW_V2)
```

Comment:
Please remove these cmake_print_variables() calls. The definitions belong elsewhere, and we really only needed to print them while we were debugging the implementation. (WITH_KRNLMON and WITH_OVSP4RT are considerably more complex, which is why they are being displayed.)